hack/ci-e2e.sh

```diff
@@ -47,6 +47,8 @@ minikube image load quay.io/metal3-io/baremetal-operator:e2e
 # Create libvirt domain
 VM_NAME="bmo-e2e-0"
 export BOOT_MAC_ADDRESS="00:60:2f:31:81:01"
+SERIAL_LOG_DIR="${REPO_ROOT}/test/e2e/_artifacts/logs/baremetal-operator-system/qemu/serial"
```

Comment:
I would interpret `baremetal-operator-system` as the namespace here and it is a bit weird to have the serial logs under this folder. Perhaps just put it directly under logs?
```suggestion
SERIAL_LOG_DIR="${REPO_ROOT}/test/e2e/_artifacts/logs/qemu/serial"
```